src/olympia/abuse/tests/test_models.py

```diff
@@ -1153,3 +1252,27 @@ def test_create_cinder_policy_with_duplicate_uuid(self):
                 text='Policy 2 Description',
                 uuid=existing_policy.uuid,
             )
+
+    def test_without_parents_if_their_children_are_present(self):
+        parent = CinderPolicy.objects.create(
+            name='Parent of Policy 1',
+            text='Policy Parent 1 Description',
+            uuid='some-parent-uuid',
+        )
+        child_policy = CinderPolicy.objects.create(
+            name='Policy 1',
+            text='Policy 1 Description',
+            uuid='some-child-uuid',
+            parent=parent,
+        )
+        lone_policy = CinderPolicy.objects.create(
+            name='Policy 2',
+            text='Policy 2 Description',
+            uuid='some-uuid',
+        )
+        qs = CinderPolicy.objects.all()
+        assert set(qs) == {parent, child_policy, lone_policy}
+        assert set(qs.without_parents_if_their_children_are_present()) == {
+            child_policy,
+            lone_policy,
+        }
```

Comment:
can we add a `.exclude(id=child_policy.id)` after to confirm that `parent` would be there if it's children weren't?